contributing-guides/translation-templates/subcommand-mention.md

```diff
@@ -90,7 +90,9 @@ Kami mempunyai dokumentasi terpisah untuk menggunakan subperintah seperti `examp
 ---
 ### it
 
-Not translated yet.
+```markdown
+Alcuni comandi aggiuntivi, come `esempio di un comando`, hanno la propria documentazione.
```

Comment:
Since this the command name which won't be translated we leave it as is.
```suggestion
Alcuni comandi aggiuntivi, come `example command`, hanno la propria documentazione.
```